hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusCreateTopicHandler.java

```diff
@@ -106,6 +111,13 @@ public void handle(@NonNull final HandleContext handleContext) {
             builder.submitKey(op.submitKey());
         }
 
+        if (op.hasAutoRenewAccount()) {
```

Comment:
It would be strongly preferable to re-use [`ExpiryValidatorImpl.resolveCreateAttempt()`](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/validation/ExpiryValidatorImpl.java#L66) here.

But if it's overly tricky for some reason, this is ok too. 😁